pkg/flavor/kubernetes/nfs.go

```diff
@@ -145,6 +153,54 @@ func (flavor *Flavor) CreateNFSVolume(pvName string, reqVolSize int64, parameter
 	}, false, nil
 }
 
+func (flavor *Flavor) CreateNFSConfigMap(nfsNamespace string) error {
+	log.Tracef(">>>>> CreateNFSConfigMap with namespace %s", nfsNamespace)
+	defer log.Tracef("<<<<< CreateNFSConfigMap")
+
+	nfsGaneshaConfig := `
+NFS_Core_Param
+{
+  NFS_Protocols= 4;
+  NFS_Port = 2049;g
+  fsid_device = false;
+}
+
+EXPORT
+{
+  Export_Id = 716;
+  Path = /export;
+  Pseudo = /export;
+  Access_Type = RW;
+  Squash = No_Root_Squash;
+  Transports = TCP;
+  Protocols = 4;
+  SecType = "sys";
+  FSAL {
+	  Name = VFS;
+  }
+}`
```

Comment:
Why don't we allow a user to specify this rather than generating it ourselves?  I thought we had discussed pushing some of those configuration options onto the user.

Reply:
yes..the challenge is it has to be in every namespace where the pod is being deployed..so might get confusing. This way, we still automatically create a configmap if one doesn't exist, and user can edit it to tweak later and restart NFS pods. We don't have to re-spin NFS images for changes then. thoughts?

Reply:
I guess I missed the check for one already existing.  Is that in the PR?

Reply:
yes..we attempt create everytime, and if error is type IsAlreadyExists we return success.

Reply:
Super! I missed that.  Thanks.